src/app/onion/dashboard/dashboard-item/dashboard-item.component.ts

```diff
@@ -160,11 +160,19 @@ export class DashboardItemComponent implements OnChanges {
           LearningObject.Status.RELEASED,
           {
             tip:
-              'This learning object is published to the ' + 
+              'This learning object is published to the ' +
                 (val ? val.name : '') +
               ' collection and can be browsed for.'
           }
         ],
+        [
+          LearningObject.Status.PROOFING,
+          {
+            tip:
+              'This learning object is currently undergoing proofing by the editorial team. ' +
+              'It is not yet published and cannot be edited until this process is complete.'
```

Comment:
We should use the "released" terminology instead of published
```suggestion
              'It is not yet released and cannot be edited until this process is complete.'
```